### DISCUSSION TITLE
Recently cooked food

### DISCUSSION THREAD

**Discussion by kazutomo2019** [NAIST-SE/AttractingContributors#532] [General]
Please tell me your own cooked food in the last week!

URL:https://github.com/kazutomo2019/Recently-cooked-food/blob/main/README.md

**Comment by ani-hovhannisyan** [comment]
Hi, I have added 1st food in your project, please can you do it back to mine?
Discussion https://github.com/NAIST-SE/AttractingContributors/discussions/543#discussion-4155998
Project https://github.com/ani-hovhannisyan/AttractingContributors-Motorcycle-Bicycle.git
